### PR TITLE
Allows RPD to place pipes on catwalks/girders

### LINKED ISSUE #32034: You can't place pipes on top of catwalks or girders with the RPD
I noticed that you cant place pipes on top of catwalks or girders with the RPD you can still place them but, you have to alt click the tile and click on the space tile to place them down or on a space tile next to it and drag it on top of it to wrench it down.

### PR DESCRIPTION
**Pull request by PKPenguin321**
Fixes https://github.com/tgstation/tgstation/issues/32034
Feels really hacky, but a better fix wasn't immediately obvious

### CODE REVIEW

File: code/game/objects/items/RPD.dm [hunk 1]
```diff
@@ -548,7 +548,7 @@ GLOBAL_LIST_INIT(RPD_recipes, list(
 	//make sure what we're clicking is valid for the current mode
 	var/is_paintable = (p_class == PAINT_MODE && istype(A, /obj/machinery/atmospherics/pipe))
 	var/is_consumable = (p_class == EATING_MODE && (istype(A, /obj/item/pipe) || istype(A, /obj/item/pipe_meter) || istype(A, /obj/structure/disposalconstruct)))
-	var/can_make_pipe = ((atmos_piping_mode || p_class == DISPOSALS_MODE) && isturf(A))
+	var/can_make_pipe = ((atmos_piping_mode || p_class == DISPOSALS_MODE) && (isturf(A) || istype(A, /obj/structure/lattice/catwalk))
```
Comment (ChangelingRain) [inline code/game/objects/items/RPD.dm]: this can just be lattice, since you can wrench pipes in space

Comment (PKPenguin321) [reply code/game/objects/items/RPD.dm]: I accidentally hid your review with another unrelated commit but: It should already work for space via the isturf check. The issue I'm fixing concerns catwalks and girders specifically

**Review by duncathan — APPROVED** [review]
fine for now

**Comment by PKPenguin321** [comment]
pls give fix tag for gbp

**Pull request merged**